presto-thrift-connector-api/src/main/java/com/facebook/presto/connector/thrift/api/utils/NameValidationUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.connector.thrift.api.utils;
```

Comment:
It's used in `ThriftMetadata` only to verify the list of schemas returned. It's a list of strings, so this check cannot be pushed to API. I can ignore this check for schemas, because it'll anyway be checked later when getting tables. Then I can remove "utils" package.